test/scripts/integration-tests.sh

```
@@ -0,0 +1,157 @@
+    fi
+
+    # Return only the PID, nothing else
+    echo "$pid"
+}
+
+# Function to stop the server
+stop_server() {
+    local pid=$1
+    if [[ -n "$pid" && "$pid" =~ ^[0-9]+$ ]]; then
+        echo "Stopping moclojer server (PID: $pid)..."
+        kill $pid || true
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)